packages/schemas/src/foundations/jsonb-types/hooks.ts

```diff
@@ -74,7 +120,34 @@ type ApiMethod = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH';
  * Define the hook event that should be triggered when the management API is called.
  */
 export const managementApiHooksRegistration = Object.freeze({
+  'POST /users': 'User.Created',
+  'DELETE /users/:userId': 'User.Deleted',
+  'PATCH /users/:userId': 'User.Info.Updated',
```

Comment:
i doubt if we need a `.Info` here since it's clear enough if we also have `User.Created` and `User.Deleted`. instead, i suppose a middle name or suffix is more useful for other possible complex events, e.g. entity + relation update